autoload/neomake/makers/ft/chef.vim

```diff
@@ -0,0 +1,35 @@
+function! neomake#makers#ft#chef#SupersetOf() abort
+    return 'ruby'
+endfunction
+
+function! neomake#makers#ft#chef#foodcritic() abort
+    return {
+      \ 'errorformat': 'FC%n: %m: %f:%l',
```

Comment:
You can just use `%W` here to make all lines warnings, instead of using "postprocess".

Can you provide some example output lines, please?